SMCalloutView.m

```diff
@@ -159,6 +159,9 @@ - (CGFloat)leftAccessoryVerticalMargin {
         return 0;
 }
 
+#pragma clang diagnostic push
```

Comment:
Instead of this, could you change `MIN` to `fminf`? I didn't realize `MIN` and `MAX` were uncool now.